### PR TITLE
Fix issue #185

### PR DESCRIPTION
**Pull request by dixoncx**
Another attempt to fix issue #185

### CODE REVIEW

File: beets/importer.py [hunk 1]
```diff
@@ -1327,7 +1327,24 @@ def resolve_duplicates(session, task):
             log.debug(u'found duplicates: {}'.format(
                 [o.id for o in found_duplicates]
             ))
-            session.resolve_duplicate(task, found_duplicates)
+
+            # Get the default action to follow from config.
+            duplicate_action = config['import']['duplicate_action'].get()
```
Comment (sampsyo) [inline beets/importer.py]: In the previous PR, you (cleverly) used `as_choice` to validate the configuration, which should be one of `skip`, `keep`, `remove`, and `ask`. Can we restore that?

**Comment by sampsyo** [comment]
This looks awesome! Thanks for moving the check to the generic importer module. :sparkles: See one small suggestion above.

**Comment by dixoncx** [comment]
I agree with your suggestion, made the changes..
Will this be fine ? Anything else ?

**Comment by sampsyo** [comment]
Wonderful! Thank you again for tackling this!

**Pull request merged**